package.json

```diff
@@ -222,11 +219,6 @@
                 }
             }
         },
-        "breakpoints": [
```

Comment:
you can leave breakpoints in, it will work with any debugging extension

Reply:
Oh ok thanks.